api/lib/application/user-orga-settings/user-orga-settings-controller.js

```diff
@@ -16,5 +16,9 @@ module.exports = {
     const result = await usecases.createUserOrgaSettings({ userId, organizationId });
 
     return h.response(userOrgaSettingsSerializer.serialize(result)).created();
+  },
+
+  update() {
+    return null;
```

Comment:
Ici pourquoi pas mettre un commentaire ou throw not implemented, pour bien faire comprendre que ça sera implémenté sur le commit suivant.

Reply:
Done